### ISSUE TITLE
RemovedInDjango30Warning: static()

### ISSUE THREAD

**Issue by rixx** [torchbox/django-libsass#31]
Apparently, the import path of the static() method changes in Django – please adjust django-libsass accordingly (not only to be future-proof, but also to save me from the amount of warnings my Django projects produce atm :wink: )

```
django_libsass.py:19: RemovedInDjango30Warning: django.contrib.staticfiles.templatetags.static() is deprecated in favor of django.templatetags.static.static()
```

**Comment by rixx** [comment]
Heads up: Django 3.0 will be the next Django release. It would be great to see django-libsass compatible.

**Comment by gasman** [comment]
Fixed in d17a524fb9e17f0b2999b27c22e8753e188ec772
